hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/helpers/KeyValueContainerUtil.java

```diff
@@ -313,16 +314,30 @@ private static void populateContainerMetadata(
 
     // Set pending deleted block count.
     final long blockPendingDeletion;
+    long blockPendingDeletionBytes = 0L;
+    Long pendingDeletionBlockBytes = metadataTable.get(kvContainerData
+        .getPendingDeleteBlockBytesKey());
     Long pendingDeleteBlockCount =
         metadataTable.get(kvContainerData
             .getPendingDeleteBlockCountKey());
-    if (pendingDeleteBlockCount != null) {
+    if (pendingDeleteBlockCount != null && pendingDeletionBlockBytes != null) {
```

Comment:
there might be case that, pendingDeleteBlockCount is not null but pendingDeletionBlockBytes is null